File: Python3/Secuenciales/ej4/ej4.py
# -*- coding: utf-8 -*-
#Dados dos números, mostrar la suma, resta, división y multiplicación de ambos.
#pedir numeroA
#almacenar en variable numeroA
#pedir numeroB
#mostrar numeroA + numeroB
#mostrar numeroA - numeroB
#mostrar numeroA * numeroB
#mostrar numeroA / numeroB

print("Mostrar + - * / de dos números")
numeroA = float(input("Introduce número A> "))
numeroB = float(input("Introduce número B> "))
print("Suma> "+str(numeroA+numeroB))
print("Resta> "+str(numeroA-numeroB))
print("Producto> "+str(numeroA*numeroB))
if(numeroB!=0):
	print("Cociente> "+str(numeroA/numeroB))
else:
	print("no se puede dividir entre 0")

File: Java/Secuenciales/ej4.java
import java.util.Scanner; //Importar la clase Scanner 

/**
*Dados dos números, mostrar la suma, resta, división y multiplicación de ambos.
*@author alberto Miguel Martinez Jimenez 
*/

//Almacenar numero1
//Almacenar numero2
//mostrar numero1 + numero2
//mostrar numero1 - numero2
//mostrar numero1 * numero2
//mostrar numero1 / numero2

public class ej4 {
	public static void main(String[] arg) {
		double numero1;
		double numero2;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner
		
		System.out.println("Introduce un numero> ");
		numero1 = scanner.nextDouble();
		
		System.out.println("Introduce un numero> ");
		numero2 = scanner.nextDouble();
		
		System.out.println(numero1+" + "+numero2+" = "+numero1+numero2);
		System.out.println(numero1+" - "+numero2+" = "+numero1-numero2);
		System.out.println(numero1+" X "+numero2+" = "+numero1*numero2);
		System.out.println(numero1+" / "+numero2+" = "+numero1/numero2);
		
		
	}
}
